.github/workflows/ci-cd-workflow.yml

```diff
@@ -221,6 +221,10 @@ jobs:
       - name: Deploy Amazon ECS task definition
         run: aws ecs update-service --cluster trainee-preprod --service ${{ github.event.repository.name }} --task-definition ${{ github.event.repository.name }}
 
+      - name: Log in to Amazon ECR
```

Comment:
This should be moved to a separate build step instead of part of the trainee deploy, since it relies on reval deploy too.

Reply:
It is updated. Please have a look. Thanks.